### PR TITLE
Update DCL

### LINKED ISSUE #129: Update DCL
Update DCL following https://github.com/c4dt/showcase/blob/master/updates/2021-02.md and add changes to log

### PR DESCRIPTION
**Pull request by tharvik**
Fix #129.

* mostly adding conferences
* move some `tech_desc` to `layman_desc` as the text seems easy enough, étoffing some
* rm "Robust RDMA", I found nothing on it
* add "ASCYLIB + OPTIK"
* for the papers, I prefered to find theses on https://infoscience.epfl.ch and when it is published at a conference, to use their main website

### CODE REVIEW

File: data/DCL/projects.yaml [hunk 1]
```diff
@@ -188,17 +206,28 @@ projects:
...
+        date_updated: 2021-02-09
 
     fleet:
-        name:         Fleet
-        description:  Machine Learning on Android
+        name: FLeet
+        description: Federated Machine Learning on Android
         layman_desc: >
-            MobNet is a framework for machine learning on android devices.
+            MobNet is a framework for federated machine learning on android
```
Comment (cgrigis) [inline data/DCL/projects.yaml:224]:
```suggestion
            MobNet is a framework for federated machine learning on Android
```

File: data/DCL/projects.yaml [hunk 2]
```diff
@@ -188,17 +206,28 @@ projects:
...
+            devices. It consumes way less battery than Standard FL while
```
Comment (cgrigis) [inline data/DCL/projects.yaml:225]:
```suggestion
            devices. It consumes much less battery than Standard FL while
```

File: data/DCL/projects.yaml [hunk 3]
```diff
@@ -188,17 +206,28 @@ projects:
...
+            increasing the model's accuracy three times. It allows for android
```
Comment (cgrigis) [inline data/DCL/projects.yaml:226]:
```suggestion
            increasing the model's accuracy three times. It allows for Android
```

File: data/DCL/projects.yaml [hunk 4]
```diff
@@ -288,30 +350,56 @@ projects:
         language: Python / C++
         information:
             - type: Paper
-              url: https://mlsys.org/Conferences/2019/doc/2019/54.pdf
-              title: "AggregaThor: Byzantine Machine Learning via Robust Gradient Aggregation (SysML19')"
+              url: https://infoscience.epfl.ch/record/265684
+              title: "AGGREGATHOR: Byzantine Machine Learning via Robust Gradient Aggregation"
```
Comment (cgrigis) [inline data/DCL/projects.yaml:354]: I don't want to get into a typography war :-), but shouldn't the large and small caps used in the paper translate to upper and lower case when using a font without small caps?

File: data/DCL/projects.yaml [hunk 5]
```diff
@@ -29,18 +29,18 @@ projects:
         code:
             type: Lab Github
             url:  https://github.com/LPD-EPFL/Garfield
-            date_last_commit: 2021-01-20
+            date_last_commit: 2021-02-04
         language: Pyton, Cuda, C++
```
Comment (cgrigis) [inline data/DCL/projects.yaml:33]:
```suggestion
        language: Python, Cuda, C++
```

**Review by cgrigis — CHANGES_REQUESTED** [review]
Minor suggestions, otherwise LGTM

File: data/DCL/projects.yaml (hunk 4 above)
Comment (tharvik) [reply data/DCL/projects.yaml:354]: Hoo, you're right, I copied it from the summary page on infoscience.epfl.ch, but didn't looked directly at the paper.

PS: typography war are one of the few worth fighting, and you easily won this one :)

**Review by cgrigis — APPROVED** [review]
Thank you!

@ineiti Do you wish to review or should we just merge?

**Comment by ineiti** [comment]
Please let me review it, this forces me to actually read the updates ;) Else I'll just forget. But if I know you're waiting on me, I'll look at it...

I would squash the commits, what do you think?

**Pull request merged**